File: Backend/src/main/java/org/example/backend/dto/TeacherSceneDto.java
package org.example.backend.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.Value;

import java.io.Serializable;

/**
 * DTO for {@link org.example.backend.domain.TeacherScene}
 */
@Data
@NoArgsConstructor
public class TeacherSceneDto implements Serializable {
    TeacherSceneIdDto id;
    TeacherDto teacher;
    SceneDto scene;
}

File: Frontend/src/app/dto/teacher-scene-dto.ts
import {TeacherSceneIdDto} from "./teacher-scene-id-dto";
import {TeacherDto} from "./teacher-dto";
import {SceneDto} from "./scene-dto";

export interface TeacherSceneDto {
  id : TeacherSceneIdDto
  teacher : TeacherDto
  scene : SceneDto
}
